examples/release_highlights/plot_release_highlights_1_1_0.py

```diff
@@ -0,0 +1,148 @@
+# flake8: noqa
+"""
+=======================================
+Release Highlights for scikit-learn 1.1
+=======================================
+
+.. currentmodule:: sklearn
+
+We are pleased to announce the release of scikit-learn 1.1! Many bug fixes
+and improvements were added, as well as some new key features. We detail
+below a few of the major features of this release. **For an exhaustive list of
+all the changes**, please refer to the :ref:`release notes <changes_1_1>`.
+
+To install the latest version (with pip)::
+
+    pip install --upgrade scikit-learn
+
+or with conda::
+
+    conda install -c conda-forge scikit-learn
+
+"""
+
+# %%
+# Quantile loss in :class:`ensemble.HistGradientBoostingRegressor`
+# ----------------------------------------------------------------
+# :class:`ensemble.HistGradientBoostingRegressor` can model quantiles with
+# `loss="quantile"` and the new parameter `quantile`.
+from sklearn.datasets import make_regression
+from sklearn.ensemble import HistGradientBoostingRegressor
+import pandas as pd
+
+X, y = make_regression(n_features=4, n_informative=8, noise=10, random_state=0)
+
+quantiles = [0.05, 0.5, 0.95]
+hist_quantiles = [
+    HistGradientBoostingRegressor(loss="quantile", quantile=quantile).fit(X, y)
+    for quantile in quantiles
+]
+predictions = {
+    f"quantile_{quantile:0.2f}": hist.predict(X)
+    for quantile, hist in zip(quantiles, hist_quantiles)
+}
+pd.DataFrame(predictions).iloc[:5]
+
+# %%
+# `get_feature_names_out` Avaliable in All Transformers
+# -----------------------------------------------------
+# :term:`get_feature_names_out` is now avaliable in all Transformers. This enables
```

Comment:
```suggestion
# :term:`get_feature_names_out` is now available in all Transformers. This enables
```

There is a similar typo in the title as well `get_feature_names_out` Avaliable in All Transformers (I did not do a suggestion since @lorentzenchr already created one one the same line about a different thing)